app/src/components/MultipleScheduleDisplay.tsx

```diff
@@ -23,8 +23,8 @@ export default function MultipleScheduleDisplay(props: Props) {
                 </b>
             </p>
 
-            {schedulesToShow.map((schedule: Schedule, i: number) => (
-                <div>
+            {schedulesToShow.map((schedule: Schedule, i) => (
+                <div key={i}>
```

Comment:
Same reason.

Reply:
I think it's OK to use index as a key here because schedulesToShow never changes (we are not deleting or inserting anything in the array).